### ISSUE TITLE
Not exactly sure what happeend

### ISSUE THREAD

**Issue by Dallas1117** [ldtteam/minecolonies#5706]
Basically I was upgrading my citizen huts to level 4 before I logged off 2 days ago. I got on today and saw structurize and minecolonies were both updated (1.16.1) and I updated. I went into game and walked over to my builder. He was working like normal then I saw he was over at the "House" Hut with a red gear like he wanted something so I did recall worker to the builder hut and got the error that says post my lastest.log.

Here is my lastest log:

https://gist.github.com/Dallas1117/acf98168a70cb78b547260c0aefecad7

**Comment by Raycoms** [comment]
Please fill in the complete template. This latest.log is not having any error, so it's difficult to help you then.

**Comment by Dallas1117** [comment]
- [X ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [X ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.1
- MineColonies Version: 0.13.213
- Structurize Version: 0.13.61
- Related Mods and their Versions:
Not Sure.

## Expected behavior

Builder upgrading hut. (he was before updating)
## Actual behavior

Listed above in my first comment. 

https://gyazo.com/cb158bd6654bcdbb8aa72d262c0f55cd
## Steps to reproduce the issue
Listed above is exactly what I did.



1.  Maybe start an upgrade then go update the files? 
2.  The builder had everything he needed and went to work on the hut.
3.  He went to the House to start upgrading and got the error.

### Logs


https://gist.github.com/Dallas1117/acf98168a70cb78b547260c0aefecad7

- latest.log: 
- crashlog: 

### Notes

Basically I was upgrading my citizen huts to level 4 before I logged off 2 days ago. I got on today and saw structurize and minecolonies were both updated (1.16.1) and I updated. I went into game and walked over to my builder. He was working like normal then I saw he was over at the "House" Hut with a red gear like he wanted something so I did recall worker to the builder hut and got the error that says post my lastest.log.
---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.
